### PR TITLE
Status Indicators - RSC-1060

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1060

### CODE REVIEW

File: lib/src/components/NxStatusIndicator/NxStatusIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+
+import withClass from '../../util/withClass';
+import { Props } from './types';
+
+const NxStatusIndicator = withClass('span', 'nx-status-indicator', 'status');
+
+function mkModifiedStatusIndicator(modifier: string) {
```
Comment (entroform) [inline lib/src/components/NxStatusIndicator/NxStatusIndicator.tsx:15]: what does "mk" in `mkModifiedStatusIndicator` mean?

**Review by entroform — APPROVED** [review]
LGTM!

File: lib/src/components/NxStatusIndicator/NxStatusIndicator.tsx (hunk 1 above)
Comment (rpokorny) [reply lib/src/components/NxStatusIndicator/NxStatusIndicator.tsx:15]: > what does "mk" in mkModifiedStatusIndicator mean?

Make

File: gallery/src/styles/NxStatusIndicator/NxStatusIndicatorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            <NxTable.Cell>Description</NxTable.Cell>
+            <NxTable.Cell>Suggested Paired Wording</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>negative</NxCode></NxTable.Cell>
+            <NxTable.Cell>The item in question is absent or not activated. </NxTable.Cell>
+            <NxTable.Cell>
+              "Off", "Disabled", "Inactive", "Not Started", "Down"
```
Comment (achen5) [inline gallery/src/styles/NxStatusIndicator/NxStatusIndicatorPage.tsx:60]: How about adding "Offline"? The status indicator could be useful for indicating whether a server is "Online" or "Offline".

File: gallery/src/styles/NxStatusIndicator/NxStatusIndicatorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>positive</NxCode></NxTable.Cell>
+            <NxTable.Cell>
+              The item in question is present and active or complete.
+            </NxTable.Cell>
+            <NxTable.Cell>
+              "On", "Enabled", "Active", "Complete", "Up"
```
Comment (achen5) [inline gallery/src/styles/NxStatusIndicator/NxStatusIndicatorPage.tsx:69]: > The status indicator could be useful for indicating whether a server is "Online" or "Offline".

How about adding "Online" here?

**Review by achen5 — APPROVED** [review]
Overall, LGTM!

**Pull request merged**